src/app/pages/observations/observation-detail.component.html

```diff
@@ -331,7 +331,25 @@ <h3>{{'List of evidence' | translate}}</h3>
             <p *ngIf="!documents.length">{{'No evidence' | translate}}</p>
           </div>
           <div *ngIf="!isDisabled()" class="small-12 medium-6 columns">
-            <!-- 9.2.1 Evidence -->
+            <!-- 9.2.1 Type -->
+            <div [ngClass]="{ 'form-group': true, '-required': !!evidence.attachment || !documents.length }">
+              <label for="evidence_type">{{'observation.evidenceType.title' | translate}}</label>
+              <select id="evidence_type" name="evidence_type" [(ngModel)]="evidence.type" #evidence_type="ngModel" [required]="!!evidence.attachment || !documents.length">
+                <option [value]="null" disabled>{{'observation.evidenceType.select' | translate}}</option>
```

Comment:
`evidence.type` is not `null` initially so “Please select an evidence type” is never shown.

You need to set its value to `null`.